kdc/krb5tgs.c

```diff
@@ -2416,7 +2416,8 @@ _kdc_tgs_rep(astgs_request_t r)
 				 r->armor_crypto,
 				 &req->req_body,
 				 ret, r->e_text,
-				 NULL, NULL,
+				 ticket->client,
+				 ticket->server,
```

Comment:
`ticket` may be NULL here, so you may wish to avoid dereferencing it in that case. Something like:
```
ticket ? ticket->client : NULL,
ticket ? ticket->server : NULL,
```

Reply:
ah thank you, will fix